src/transformers/modeling_tf_utils.py

```diff
@@ -348,6 +424,15 @@ def input_processing(func, input_ids, **kwargs):
     if "kwargs" in output:
         del output["kwargs"]
 
+    boolean_dict = {k: v for k, v in output.items() if k in boolean_properties}
```

Comment:
(nit) I'd actually move ["return_dict", "output_attentions", "output_hidden_states", "use_cache"] directly into the line here -> replace boolean_properties with ["return_dict", "output_attentions", "output_hidden_states", "use_cache"] -  it'd make it easier to read for me